src/javascript/app/common/form_validation.js

```diff
@@ -111,7 +111,11 @@ const Validation = (() => {
                         });
 
                         if (field.$.attr('type') === 'password') {
-                            field.$.next('#password_toggle')[0].onclick = () => togglePasswordVisibility(field);
+                            const [password_toggle] = field.$.next('#password_toggle');
+                            if (password_toggle) {
```

Comment:
```suggestion
                            password_toggle?.onclick = () => togglePasswordVisibility(field);
```

Reply:
I think the current implementation is more readable than that, but up to you.

Reply:
I'm not sure if option chaining works in `binary-static` currently.